pharmd-app/src/components/Fields/NoteField.js

```diff
@@ -37,37 +34,38 @@ const Title = styled.h3`
 
 const Date = styled.h4`
   font-weight: normal;
-  font-size: .85em;
-  margin: 0px 0px 0px .5em;
+  font-size: 0.85em;
+  margin: 0px 0px 0px 0.5em;
   color: ${props => props.theme.palette.tertiary.main};
 `;
 
 const Content = styled.div`
   height: 3em;
   line-height: 1em;
   overflow: hidden;
-  margin-top: .4em;
+  margin-top: 0.4em;
   word-break: break-all;
-`
+`;
 
 const NoteField = () => {
-    return (
-        <Info>
-            <Heading>
-                <Title>Note Title</Title>
-                <NoteIcon src={EditIcon} color="black" size="small" isPrimary={"primary"}/>
-            </Heading>
-            <Time>
-                <NoteIcon src={AccessTimeIcon} color="grey" size="inherit" />
-                <Date>Date</Date>
-            </Time>
-            {/* if the length of string is more than 2 lines - ask jose how to check for this*/}
```

Comment:
@Jose-S I think it would be useful to explain this in the documentation of this file if possible